annotated-projects/PXD005241/PXD005241.sdrf.tsv

```diff
@@ -0,0 +1,19 @@
+Sample 3	Oryza sativa	Leaf	Callus	Root	Shoot meristem	Young panicles	Mature panicles	not applicable	not applicable	3	run 15	 japonica	PXD005241	Rice_phos_root_TiO2_25per_F3_R1.raw	https://www.ebi.ac.uk/pride/archive/projects/PXD005241/Rice_phos_root_TiO2_25per_F3_R1.raw	1	1	NT=label free sample	AC=MS:1001911;NT=Q Exactive	AC=MS:1001251;NT=Trypsin	NT=Carbamidomethyl;AC=UNIMOD:4;TA=C;MT=Fixed	NT=Oxidation;MT=Variable;TA=M;AC=UNIMOD:47	NT=Acetyl;MT=Variable;PP=Protein N-term;AC=UNIMOD:13	NT=Phospho;AC=UNIMOD:21;TA=S;MT=Variable	NT=Phospho;AC=UNIMOD:21;TA=T;MT=Variable	NT=Phospho;AC=UNIMOD:21;TA=Y;MT=Variable	not available	not available
+Sample 1	Oryza sativa	Leaf	Callus	Root	Shoot meristem	Young panicles	Mature panicles	not applicable	not applicable	1	run 16	 japonica	PXD005241	Rice_phos_young panicles_TiO2_25per_F1_R1.raw.raw	https://www.ebi.ac.uk/pride/archive/projects/PXD005241/Rice_phos_young panicles_TiO2_25per_F1_R1.raw.raw	1	1	NT=label free sample	AC=MS:1001911;NT=Q Exactive	AC=MS:1001251;NT=Trypsin	NT=Carbamidomethyl;AC=UNIMOD:4;TA=C;MT=Fixed	NT=Oxidation;MT=Variable;TA=M;AC=UNIMOD:48	NT=Acetyl;MT=Variable;PP=Protein N-term;AC=UNIMOD:14	NT=Phospho;AC=UNIMOD:21;TA=S;MT=Variable	NT=Phospho;AC=UNIMOD:21;TA=T;MT=Variable	NT=Phospho;AC=UNIMOD:21;TA=Y;MT=Variable	not available	not available
+Sample 2	Oryza sativa	Leaf	Callus	Root	Shoot meristem	Young panicles	Mature panicles	not applicable	not applicable	2	run 17	 japonica	PXD005241	Rice_phos_young panicles__TiO2_25per_F2_R1.raw.raw	https://www.ebi.ac.uk/pride/archive/projects/PXD005241/Rice_phos_young panicles__TiO2_25per_F2_R1.raw.raw	1	1	NT=label free sample	AC=MS:1001911;NT=Q Exactive	AC=MS:1001251;NT=Trypsin	NT=Carbamidomethyl;AC=UNIMOD:4;TA=C;MT=Fixed	NT=Oxidation;MT=Variable;TA=M;AC=UNIMOD:49	NT=Acetyl;MT=Variable;PP=Protein N-term;AC=UNIMOD:15	NT=Phospho;AC=UNIMOD:21;TA=S;MT=Variable	NT=Phospho;AC=UNIMOD:21;TA=T;MT=Variable	NT=Phospho;AC=UNIMOD:21;TA=Y;MT=Variable	not available	not available
+Sample 3	Oryza sativa	Leaf	Callus	Root	Shoot meristem	Young panicles	Mature panicles	not applicable	not applicable	3	run 18	 japonica	PXD005241	Rice_phos_young panicles__TiO2_25per_F3_R1.raw.raw	https://www.ebi.ac.uk/pride/archive/projects/PXD005241/Rice_phos_young panicles__TiO2_25per_F3_R1.raw.raw	1	1	NT=label free sample	AC=MS:1001911;NT=Q Exactive	AC=MS:1001251;NT=Trypsin	NT=Carbamidomethyl;AC=UNIMOD:4;TA=C;MT=Fixed	NT=Oxidation;MT=Variable;TA=M;AC=UNIMOD:50	NT=Acetyl;MT=Variable;PP=Protein N-term;AC=UNIMOD:16	NT=Phospho;AC=UNIMOD:21;TA=S;MT=Variable	NT=Phospho;AC=UNIMOD:21;TA=T;MT=Variable	NT=Phospho;AC=UNIMOD:21;TA=Y;MT=Variable	not available	not available

```

Comment:
The source name of different samples should be different.  The source name here should be Sample 1, Sample 2, Sample 3, Sample 4, Sample 5, Sample 6, Sample 7...

Reply:
@daichengxin thanks, I have changed it.